### DISCUSSION TITLE
v3 config for lualine

### DISCUSSION THREAD

**Discussion by carlesba** [folke/trouble.nvim#471] [Q&A]
I'm adding the config suggested in docs adapted for the `config` method in LazyVim as my lualine setup is done there.

Here's my config:

<img width="582" alt="Screenshot 2024-06-02 at 19 35 59" src="https://github.com/folke/trouble.nvim/assets/1178459/108bc751-2198-4e5d-9fb8-df16323eeb88">
<img width="242" alt="Screenshot 2024-06-02 at 19 35 50" src="https://github.com/folke/trouble.nvim/assets/1178459/c6336fd0-5780-4082-955c-18753b6340ef">

This is the output I'm getting:

<img width="877" alt="Screenshot 2024-06-02 at 19 35 42" src="https://github.com/folke/trouble.nvim/assets/1178459/281621e6-a1bf-4754-9790-87f9acda3f3e">

is this what trouble is expected to show in my statusline? How can I read this? How can I fix my config?

**Comment by k14lb3** [comment]
Try `mode = "symbols"`.

**Reply by carlesba** [reply]
didn't fix it

**Comment by folke** [comment]
The one in Lazyvim filters out a lot of symbol types...

**Reply by carlesba** [reply]
is that the expected output? What am I doing wrong?
![image](https://github.com/folke/trouble.nvim/assets/1178459/79c95a66-3a29-4b60-8436-b4f648e03636)
